CMakeLists.txt

```diff
@@ -25,5 +29,19 @@ if(BUILD_TESTING)
 endif()
 
 add_subdirectory(src/beman/exemplar)
-
 add_subdirectory(examples)
+
+write_basic_package_version_file(
```

Comment:
Inspect current behavior from https://github.com/beman-project/execution26/blob/main/CMakeLists.txt and https://github.com/beman-project/optional26/blob/main/CMakeLists.txt. 

I tried to extract only the minimum required code to do the install.

Expected tree:

```bash
@pc:~/git/Beman/exemplar$ cmake --install build --prefix /opt/beman.exemplar
-- Install configuration: ""
-- Installing: /opt/beman.exemplar/lib/libbeman.exemplar.a
-- Installing: /opt/beman.exemplar/include
-- Installing: /opt/beman.exemplar/include/beman
-- Installing: /opt/beman.exemplar/include/beman/exemplar
-- Installing: /opt/beman.exemplar/include/beman/exemplar/identity.hpp
-- Installing: /opt/beman.exemplar/lib/cmake/beman.exemplarConfig.cmake
-- Installing: /opt/beman.exemplar/lib/cmake/beman.exemplarConfigVersion.cmake
dariusn@pc:~/git/Beman/exemplar$ tree /opt/beman.exemplar
/opt/beman.exemplar
├── include
│   └── beman
│       └── exemplar
│           └── identity.hpp
└── lib
    ├── cmake
    │   ├── beman.exemplarConfig.cmake
    │   └── beman.exemplarConfigVersion.cmake
    └── libbeman.exemplar.a

6 directories, 4 files
```

I would like your feedback and decide a standard pattern, which will be applied initially to beman.examplar, and eventually mirror it into other 2 repos.

Reply:
> [W]e should NOT export...docs

I'm okay with this for now. Eventually, we should export docs at installation time to the `docs/beman.short_name/` directory, but we need to get our docs story figured out first.

> @camio proposed to add a subdirectory for cmake configs

Yes, this is the convention for the ArchLinux distribution and probably others as well. See [openjpeg2's contents](https://archlinux.org/packages/extra/x86_64/openjpeg2/) for an example. [Config Mode's Search Procedure](https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure) includes a table indicating how *Config.cmake files are found.